src/Entity/FileEntity.php

```diff
@@ -83,6 +83,16 @@ public function preSave(EntityStorageInterface $storage) {
     if ($this->bundle() === FILE_TYPE_NONE) {
       $this->updateBundle();
     }
+    // If the original file entity is empty.
```

Comment:
Comment is not correct, *not* empty. Also, would be easier to understand as a single sentence and continue on the same line.